## The DevoWorm Project: raising the worm with data

Bradly Alicea<sup>1</sup>, Stephen Larson<sup>2</sup>, Steve McGrew<sup>3</sup>, Richard Gordon<sup>4</sup>

<sup>1</sup>Crop Sciences, University of Illinois Urbana-Champaign, balicea@openworm.org; <sup>2</sup>MetaCell, La Jolla CA USA; <sup>3</sup> New Light Industries, Ltd., Spokane, WA USA; <sup>4</sup>Gulf Specimen Aquarium & Marine Lab, Panacea, FL

DevoWorm [1] is a distributed collaboration and associated with the OpenWorm project [2]. DevoWorm is focused on leveraging meta-analysis and computational modeling to better understand C. elegans embryogenesis by synthesizing the existing knowledge-base. We will provide a broad overview of the project goals, which conssist of three parts:

1) Assembly/integration of existing datasets: DevoWorm will use cellular-level data (position, radius, mother and neighboring cells, expressed genes) to build phenomenological, mean-field [3] and dynamic models starting from early embryogenesis. Building developmental emulations from this wealth of community data will have wide-ranging implications.

2) Higher-level data/computational modeling: DevoWorm will derive higher-level descriptions of the C. elegans embryo including annotation via higher-level markup languages (SBML, RDF), with biophysical modeling using CompuCell3D [4]. Secondary datasets of new created variables will graphically represent developmental processes, complementing higher-level descriptions in the form of layered data analysis.

3) Theoretical investigations: It may be possible to re-interpret the classic lineage tree view of C. elegans development and explain mosaic and regulative development via a unified theory. One way this can be accomplished is through reorganization of the C. elegans lineage tree as a differentiation tree [5]. A C. elegans differentiation tree may prove useful for understanding developmental mutants, and the evolutionary relationships between various types of developmental systems.

### References
1. Alicea, B. et.al bioRxiv, doi:10.1101/009993 (2014). 

2. Szigeti, B. et.al Front Computa Neurosci, 8, 137 (2014).

3. Ellner, S.P.; Guckenheimer, J. Dynamic Models in Biology; Princeton University Press (2011).

4. Izaguirre, J.A. et.al Bioinformatics, 20, 1129-1137 (2004).

5. Gordon, R. The Hierarchical Genome and Differentiation Waves: Novel Unification of Development, Genetics and Evolution; World Scientific: Singapore (1999).
